<ui version="4.0" >
 <class>SAPartitionModelerWidget</class>
 <widget class="QWidget" name="SAPartitionModelerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Number of discretization steps</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="sliderDSteps" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QSpinBox" name="numdstepsEdit" >
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Number of states for discretization</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSlider" name="sliderStates" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QSpinBox" name="numstatesEdit" >
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Number of autocorrelation coefficients to consider</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QSlider" name="sliderAutoc" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QSpinBox" name="autoc" />
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Parameters for Simulated Annealing step</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Initial temperature</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QDoubleSpinBox" name="initTemp" >
        <property name="decimals" >
         <number>1</number>
        </property>
        <property name="singleStep" >
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Abort temperature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" >
       <widget class="QDoubleSpinBox" name="abortTemp" >
        <property name="decimals" >
         <number>6</number>
        </property>
        <property name="minimum" >
         <double>0.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.001000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="2" column="2" >
       <widget class="QDoubleSpinBox" name="alpha" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Cooldown coefficient</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QPushButton" name="computeButton" >
     <property name="text" >
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QLabel" name="progressLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderDSteps</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numdstepsEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>268</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel" >
     <x>432</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numdstepsEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderDSteps</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>421</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel" >
     <x>279</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderStates</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numstatesEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>280</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>429</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numstatesEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderStates</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>412</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel" >
     <x>276</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderAutoc</sender>
   <signal>valueChanged(int)</signal>
   <receiver>autoc</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>254</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel" >
     <x>425</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoc</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderAutoc</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>419</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel" >
     <x>278</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
